anime/components/awardspage/ActualVoting.vue

```diff
@@ -606,57 +611,41 @@ export default ({
 </script> -->
   
 <style scoped>
-.input-Box {
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  justify-content: flex-start;
-  column-gap: 1rem;
-  margin-right: 2rem;
-  width: 30rem;
-}
-.input {
-  background-color: var(--bg-secondary);
-  border-radius: 0.75rem;
-  border: none;
-  color: var(--light-text);
-  font-size: var(--h5);
-  font-weight: var(--fw-regular);
-  padding: 0.5rem 1rem;
-  width: 100%;
-  margin-top: 2rem;
-}
-.input:focus {
-  outline: none;
-}
-
 #actual-voting {
   padding-top: 9vh;
-  width: 70vw;
+  width: 75vw;
   margin: auto;
-  margin-bottom: 10vh;
+  margin-bottom: 10rem;
   color: var(--white);
 }
 .award-name {
   background: var(--secondary);
   font-size: var(--h1);
   text-align: center;
   margin-top: 7rem;
+  border-radius: 3rem;
+}
+.input {
+  color: var(--search-text);
+  font-size: var(--h4);
+  background-color: var(--search-bg);
+  border-radius: 1rem;
+  border: none;
+  outline: none;
+  padding: 1rem 2rem;
+  margin: 3rem 0 4rem;
+  width: 40%;
 }
 .nominee-container {
-  margin-top: 5vh;
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: space-between;
+  display: grid;
```

Comment:
In our case its much more easier/we dont need to use css grid. using css grid here is kinda too extra b/c we arn't doing any extreme layouts like
```
box1        box2
        box5
box3        box4
```
that example is a use case of when you would use grid because doing this layout would be very hard with flexbox/annoying. Also using flex and grid is totally fine. 

```
box1      box2    box2
box1      box2    box2
box1      box2    box2
```
the containers look like that right now and its much easier to just use flex box. Just a side note.